tests/components/utils-test.js

```diff
@@ -237,6 +237,30 @@ describe('Utils', function () {
     expect(isValidString).toBeTruthy();
   });
 
+  it('tests sanitizeConfigData', function () {
+    var data = null;
+    var sanitizedConfigData = Utils.sanitizeConfigData(data);
+    expect(sanitizedConfigData).toEqual({});
+
+    data = '';
```

Comment:
Can you add tests that assure that non empty string and numbers return something valid.